docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -14,6 +14,92 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
+- Cluster API Provider Nutanix: `v1.2.1` to `v1.2.3`
+
+### Cilium Upgrades
+- Cilium: `v1.11.15` to `v1.12.11`
+  - [Changelog](https://github.com/cilium/cilium/releases/tag/v1.12.11)
+
+
+  **Note:** If you are using the vSphere provider with the Redhat OS family, there is a known issue with VMWare and the new Cilium version that only affects our Redhat variants. To prevent this from affecting your upgrade from EKS Anywhere v0.16 to v0.17, we are adding a temporary daemonset to disable UDP offloading on the nodes before upgrading Cilium. After your cluster is upgraded, the daemonset will be deleted. This note is strictly informational as this change requires no additional effort from the user.
+
+### Changed
+- Change the default node startup timeout from 10m to 20m in Bare Metal provider [#5942](https://github.com/aws/eks-anywhere/issues/5942)
+- EKS Anywhere now fails on pre-flights if a user does not have required permissions. [#5865](https://github.com/aws/eks-anywhere/issues/5865)
+- `eksaVersion` field in the cluster spec is added for better representing CLI version and dependencies in EKS-A cluster [#5847](https://github.com/aws/eks-anywhere/issues/5847)
+- vSphere datacenter insecure and thumbprint is now mutable for upgrades when using full lifecycle API [6143]https://github.com/aws/eks-anywhere/issues/6143
+
+### Fixed
+- Fix cluster creation failure when the `<Provider>DatacenterConfig` is missing apiVersion field [#6096](https://github.com/aws/eks-anywhere/issues/6096)
+- Allow registry mirror configurations to be mutable for Bottlerocket OS [#2336](https://github.com/aws/eks-anywhere-build-tooling/pull/2336)
+- Patch an issue where mutable fields in the EKS Anywhere CloudStack API failed to trigger upgrades [#5910](https://github.com/aws/eks-anywhere/issues/5910)
+- image builder: Fix runtime issue with git in image-builder v0.16.2 binary [#2360](https://github.com/aws/eks-anywhere-build-tooling/pull/2360)
+- Bare Metal: Fix issue where metadata requests that return non-200 responses were incorrectly treated as OK [#2256](https://github.com/aws/eks-anywhere-build-tooling/pull/2256)
+
+### Known Issues:
+- Upgrading Docker clusters from previous versions of EKS Anywhere may not work on Linux hosts due to an issue in the Cilium 1.11 to 1.12 upgrade.  Docker clusters is meant solely for testing and not recommended or support for production use cases.  There is currently no fixed planned.
+- If you are installing EKS Anywhere Packages, Kubernetes versions 1.23-1.25 are incompatible with Kubernetes versions 1.26-1.27 due to an API difference. This means that you may not have worker nodes on Kubernetes version <= 1.25 when the control plane nodes are on Kubernetes version >= 1.26. Therefore, if you are upgrading your control plane nodes to 1.26, you must upgrade all nodes to 1.26 to avoid failures.
+
+### Deprecated
+- The bundlesRef field in the cluster spec is now deprecated
```

Comment:
Can we mention that it's being replaced by eksaVersion? Also, mention it will be removed three versions from now.